### PR TITLE
[PIM-6374] Move tabs to the left column

### PR DESCRIPTION
**Pull request by pierallard**
| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/column-navigation.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            events: {
+                'click .column-navigation-link': 'selectTab',
+                'click .AknDropdown-menuLink': 'selectTab'
+            },
+
+            /**
+             * {@inheritdoc}
+             */
+            configure: function () {
+                this.tabs = [];
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/column-navigation.js]: drop it

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/column-navigation.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+             * @param event
+             */
+            registerTab: function (event) {
+                this.tabs.push({
+                    code: event.code,
+                    isVisible: event.isVisible,
+                    label: event.label
+                });
+
+                this.currentTab = event.currentTab;
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/column-navigation.js]: should be done in configure

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/column-navigation.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+                this.render();
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/column-navigation.js]: Should not be done now
After configure

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/column-navigation.js [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+            },
+
+            /**
+             * Returns the current tab.
+             * If there is no selected tab, returns the first available tab.
+             */
+            getCurrentTabOrDefault: function () {
+                var result = null;
+
+                _.each(this.tabs, function (tab) {
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/column-navigation.js]: _.findWhere

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/form-tabs.js [hunk 5]
```diff
@@ -20,11 +20,11 @@ define(
             template: _.template(template),
             className: 'AknTabContainer tabbable tabs-top',
             tabs: [],
-            fullPanel: false,
             urlParsed: false,
             events: {
                 'click header ul.nav-tabs li': 'selectTab'
             },
+            currentKey: 'current_form_tab',
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/form-tabs.js]: add int to cconfiguroianferpog,feprog,

File: src/Pim/Bundle/UIBundle/Resources/public/less/components/TabContainer.less [hunk 6]
```diff
@@ -29,12 +30,15 @@
     }
   }
 
+  // @deprecated - will be removed in 1.9
```
Comment (pierallard) [inline src/Pim/Bundle/UIBundle/Resources/public/less/components/TabContainer.less]: :fire:

**Review by tamarasaurus — APPROVED** [review]

**Review by Alban44 — APPROVED** [review]

**Comment by pierallard** [comment]
Mob review with @tamarasaurus @Alban44  @phaseinducer
GTM under condition about comments

**Pull request merged**